### PR TITLE
Include conditional template properties

### PR DESCRIPTION
**Pull request by smbea**
Related to https://github.com/bpmn-io/bpmn-js-properties-panel/issues/690

* bump `bpmn-js-properties-panel`
* add conditional template properties example

### CODE REVIEW

File: resources/element-templates/cloud-samples.json [hunk 1]
```diff
@@ -180,6 +180,145 @@
...
+        "value": "",
+        "type": "String",
+        "optional": true,
+        "binding": {
+          "type": "zeebe:input",
+          "name": "body"
+        },
+        "condition": {
+          "property": "httpMethod",
+          "oneOf": ["patch, post, delete"]
```
[the remark this answers is not in the document]
Comment (smbea) [reply resources/element-templates/cloud-samples.json:243]: It should

**Comment by smbea** [comment]
> I suppose we still need to update the properties panel to test this, correct?

I'm working on it right now, and will comment here when done

File: client/package.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
     "@ibm/plex": "^6.0.0",
     "@sentry/browser": "^6.3.6",
     "bpmn-js": "^9.0.3",
-    "bpmn-js-properties-panel": "^1.0.0-alpha.12",
+    "bpmn-js-properties-panel": "^1.2.0",
```
Comment (barmac) [inline client/package.json:21]: We need to bump and release the properties panel in `camunda-bpmn-js` yet and bump it here.

**Comment by barmac** [comment]
Now it's only about a new template example on develop, and it works correctly.

**Review by barmac — APPROVED** [review]

**Comment by christian-konrad** [comment]
@smbea please note that for Web Modeler users, an array of templates does not work. In the Web Modeler, there is a 1:1 file-to-template relation. We may provide those again as multiple single files once we have this documented in the Web Modeler sections (or we have them as templates right in the Web Modeler, so we don't need to care at all)

**Pull request merged**